README-source.md

```diff
@@ -587,42 +587,27 @@ This object holds the user's auth details and the data for the API requests.
 
 ```javascript
 {
-  createdBy: '{{chef_name}}'
-  style: '{{style}}'
+  createdBy: 'his name is {{123__chef_name}}'
+  style: 'he cooks {{456__style}}'
 }
 ```
 
+> "curlies" are data mapped in from previous steps. They take the form `{{NODE_ID__key_name}}`. You'll usually want to use `bundle.inputData` instead.
+
 ### `bundle.meta`
 
-`bundle.meta` is extra information useful for doing advanced behaviors depending on what the user is doing. It looks something like this:
+`bundle.meta` is extra information useful for doing advanced behaviors depending on what the user is doing. It has the following options:
 
-```javascript
-{
-  frontend: false,
-  prefill: false,
-  hydrate: true,
-  test_poll: false,
-  standard_poll: true,
-  first_poll: false,
-  limit: -1,
-  page: 0,
-}
-```
-
-For example - if you want to do pagination - you could do:
-
-```javascript
-const getList = (z, bundle) => {
-  const promise = z.request({
-    url: 'http://example.com/api/list.json',
-    params: {
-      limit: 100,
-      offset: 100 * bundle.meta.page
-    }
-  });
-  return promise.then((response) => response.json);
-};
-```
+| key | default | description |
+| ------------- | ------- | ---------------------------------------- |
+| frontend | `false` | whether this poll was initiated via the zap editor |
+| prefill | `false` | whether this poll is being used as a dynamic dropdown |
+| hydrate | `true`  | whether the results of this poll will be hydrated |
+| test_poll | `false` | whether the poll was triggered by a user testing their account |
+| standard_poll| `true`  | the opposite of `test_poll` |
+| first_poll | `false` | whether the poll was an initial check for items when a zap is turned on (none of which will trigger) |
+| limit | `-1`    | the number of items to fetch for performance reasons. `-1` indicates there's no limit |
```

Comment:
Can probably drop the "for performance reasons"

Reply:
do we use this for anything? It truncates the results that come in, but I don't understand why we'd use that

https://github.com/zapier/zapier/blob/develop/api/resources_flow.py#L1390-L1391

Reply:
It's going to be more important when Better Sample Data launches. They'll be able to use it to say "grab 3 records" for samples in the editor, vs a -1 which might cause us to grab 100 records and throw 97 away